src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const arrMan = people.filter(a => a.sex === 'm');
+  const arrManCen = arrMan.filter(a => Math.ceil(a.died / 100) === century);
+
+  return century
+    ? arrManCen
+      .map(a => a.died - a.born)
+      .reduce((a, x) => a + x, 0) / arrManCen.length
+    : arrMan
+      .map(a => a.died - a.born)
+      .reduce((a, x) => a + x, 0) / arrMan.length;
```

Comment:
Don't repeat yourself: just check at the start of the function is century is passed, use correct filter and then call map and reduce for it